market/orderTypes/limit.ts

```
@@ -0,0 +1,54 @@
+import marketLogic from ".."
+import { getAllCollectibles, getBuyActiveOrdersByCollectibleAndType, getSellActiveOrdersByCollectibleAndType } from "../../database"
+import { Order } from "../../types/Order"
+import { OrderLogic } from "../../types/OrderLogic"
+
+
+const limit: OrderLogic = {
+    id: 'LIMIT',
+    updateAll: function() {
+        const collectibles = getAllCollectibles()
+        
+        for(let i in collectibles)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'i' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)